### PR TITLE
Update cypress

### PR DESCRIPTION
**Pull request by nicolaskempf57**
This PR :
- fixes all e2e errors due to previous build changes
- adds a one line running of fixtures, server and test run
- update cypress to latest version

We can implements cypress run on CI in a new PR or here.
Currently, this adds a `udata-front-e2e.cfg.example` file that contains non sensitive default for udata-front and empty variables for secret ones.

Closes https://github.com/etalab/udata-front/pull/212

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -5,14 +5,19 @@
...
     "postinstall": "patch-package",
     "i18n:report": "vue-i18n-extract report",
     "i18n:extract": "npm run i18n:report -- --add",
-    "start": "npm run dev",
-    "serve": "udata serve"
+    "serve": "udata serve",
+    "prepare-e2e": "UDATA_SETTINGS=${INIT_CWD}/cypress/udata-front-e2e.cfg udata generate-fixtures",
+    "serve-e2e": "UDATA_SETTINGS=${INIT_CWD}/cypress/udata-front-e2e.cfg udata serve",
+    "prepare-and-serve-e2e": "npm run prepare-e2e && npm run serve-e2e",
+    "ci": "start-server-and-test prepare-and-serve-e2e http://localhost:7000 test"
```
Comment (maudetes) [inline package.json:20]: Oh, funny this tool!

File: cypress/cypress.config.js [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+  e2e: {
+    setupNodeEvents(on, config) {
+      console.log('Starting script');
+      return getLocations(`${config.baseUrl}/sitemap.xml`).then(urls => {
+        console.log(urls);
+        config.env.URLS = urls;
+        return config;
+      });
+    },
+    baseUrl: "http://localhost:7000",
```
Comment (maudetes) [inline cypress/cypress.config.js:13]: If we want to run it locally, we may need to make it configurable (to use `dev.local` for example :p)

File: cypress/udata-front-e2e.cfg.example [hunk 3]
```diff
@@ -0,0 +1,10 @@
+PLUGINS = ['front']
+THEME = 'gouvfr'
+
+MONGODB_HOST = 'mongodb://localhost:27017/udata-front-e2e'
+POST_DISCUSSIONS_ENABLED = True
+
+CAPTCHETAT_CLIENT_ID =
```
Comment (maudetes) [inline cypress/udata-front-e2e.cfg.example:7]: Maybe adding the CI here would make us think about the way to set these? I've done something similar for ligthhouse in .circleci/config.yml

File: cypress/cypress.config.js (hunk 2 above)
Comment (nicolaskempf57) [reply cypress/cypress.config.js:13]: This is working with `npm run ci` because `dev.local:7000` is not provided in the `udata-front-e2e.cfg`.

**Pull request merged**